src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CalculatorTest {
+    private static final double DELTA = 0.01;
+    private static Calculator calculator;
+
+    @BeforeClass
+    public static void setCalculator() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    public void sumTest() {
+        Assert.assertEquals(7,calculator.result('+',2,5),DELTA);
```

Comment:
```suggestion
        Assert.assertEquals(7, calculator.result('+', 2, 5), DELTA);
```
add whitespaces to separate parameters everywhere in this class